src/components/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,71 @@
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { useMutation } from '@apollo/client';
+import { CREATE_USER_MUTATION } from '../../queries/queries';
+import './SignUpForm.css';
+
+function SignUpForm({ setUser }) {
+  const [signUpValues, setSignUpValues] = useState({
+    name: '',
+    email: '',
+    password: '',
+    passwordConfirmation: ''
+  });
+  const [createUser, { loading, error, data }] = useMutation(CREATE_USER_MUTATION);
+
+  const handleChange = (event) => {
+    const fieldInput = event.target
+    setSignUpValues({ ...signUpValues, [fieldInput.name]: fieldInput.value })
+  }
+
+  const handleSubmit = () => {
+    createUser({ variables: { name: signUpValues.name, email: signUpValues.email, password: signUpValues.password, passwordConfirmation: signUpValues.passwordConfirmation } });
```

Comment:
Suggested way to shorten this statement

```suggestion
    createUser({ variables: signUpValues });
```

Reply:
Much smarter